changelog.txt

```diff
@@ -6,6 +6,13 @@
 * Update - Remove task from the overview list for setting up multiple currencies.
 * Update - Return to task "Set up payments" after finishing KYC from WC-Admin.
 * Fix - Explicit currency formatting in customer-facing emails.
+* Fix: Update tooltip wording when deleting product variation.
+* Fix: Remove references to WooCommerce Subscriptions extension in the tooltips found on the Payment Methods settings table.
+* Fix: Update the Automatic Recurring Payments column on the Payment Methods table to only show which payment methods are supported by WooCommerce Subscriptions Core.
+* Fix: Prevent deprecation warnings when purchasing subscriptions products using WooCommerce Blocks.
+* Tweak: Update recurring payments copy on payment gateways page.
+* Fix: Incorrect text when filtering subscriptions to no results.
+* Changed: Subscription products must have a recurring amount greater than $0.
```

Comment:
Nitpick: Maybe change colons `:` to dashes `-` to match the existing changelog entries.

Reply:
Thank you for catching that, @aprea. I think that's necessary change for consistency, not a nit =). I blindly just copy-pasted and changed the copy a bit without noticing the format.

Reply:
Done in 7e6d109.